frontend/src/org/components/EventForm/steps/ParticipantsStep.module.scss

```diff
@@ -163,20 +163,29 @@ h2 {
       }
     }
 
-    .existingParticipantContainer {
+    .existingParticipantContainerWrapper {
       margin-top: 12px;
-      overflow-x: auto;
       [class$=-control]{
         border-width: 2px;
         border-color: black;
       }
-      thead th, 
-      tbody td {
-        &:nth-child(1)
-        {
-          position: sticky;
-          left: 0;
-          background-image: linear-gradient(to right, #ffffff 0%, #ffffff  99.7%, #d9d9d9 100%);
+      .existingParticipantContainer{
+        overflow-x: auto;
+        @include media.for-tablet-portrait-up {
+          thead th, 
+          tbody td {
+            &:nth-child(1)
```

Comment:
Potenciálně jde taky použít `&:first-child`, ale není to chyba.

Reply:
to vypadá líp, dík!